Example/QuranEngineApp/Classes/AppDelegate.swift

```diff
@@ -49,6 +49,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let downloadManager = container.downloadManager
         downloadManager.setBackgroundSessionCompletion(completionHandler)
     }
+    
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        // Check if the URL scheme matches the custom scheme
```

Comment:
Please remove this redundant comment. The code is clear. Same with the other inline comments below.

Reply:
addressed